### PR TITLE
Gives Nukeops a combat knife* upon spawning

### PR DESCRIPTION
**Pull request by ExpAntique**
:cl: optional name here
add: Adds a combat knife to the Nuke-ops spawning loadout

/:cl:

[why]: # Nukeops should get a cheap, default melee weapon to use when they need it. A highly trained commando shouldn't have to kick someone to death

### CODE REVIEW

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 1]
```diff
@@ -127,6 +127,7 @@
 	back = /obj/item/storage/backpack
 	ears = /obj/item/device/radio/headset/syndicate/alt
 	l_pocket = /obj/item/pinpointer/nuke/syndicate
+	2_pocket = /obj/item/melee/transforming/energy/sword/saber
```
Comment (ghost) [inline code/game/gamemodes/nuclear/nuclear.dm]: r_pocket not 2_pocket

Comment (ExpAntique) [reply code/game/gamemodes/nuclear/nuclear.dm]: oof. fixed

**Comment by Manly-Dwarf** [comment]
So in other words we're giving Nuke Ops a free 8 TC item. I could argue whether or not the Esword is actually worth 8 TC on Ops, but I think it was better when we spawned Nuke Ops with that Combat Knife since it could be attached to guns, wasn't too expensive, and could be thrown just in case. Just bring that back instead.

**Comment by JohnGinnane** [comment]
Yeah, 8TC might be much for a nukie, but giving them one is kinda too far. Did they remove combat knife and glock/stetchkin + 1 mag?

**Comment by praisenarsie** [comment]
@JohnGinnane afaik they didn't??

**Comment by ExpAntique** [comment]
@JohnGinnane the only weapon that nukies start with is a pistol with a full mag. If they want more ammo/ other shit, they have to get it from thier armoury.

 Also while the e-sword is 8 tc, it’s still not a good weapons compared the rest of the nukeop’s arsenal. Nukeops can still ditch the sword for a double-edged or some other weapon.

**Comment by ExpAntique** [comment]
+1 this comment if you want combat knife rather than e-sword 

edit: aight, whatevs

**Comment by JohnGinnane** [comment]
Should be modularised by creating a file with the same name under `hippiestation/path/to/file`

Let me know if you need help with this

**Comment by Manly-Dwarf** [comment]
>adding the garbage, bootleg /tg/ combat knife instead of our older, better, cooler combat knife

soy

**Comment by Carbonhell** [comment]
+1

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 2]
```diff
@@ -127,6 +127,7 @@
 	back = /obj/item/storage/backpack
 	ears = /obj/item/device/radio/headset/syndicate/alt
 	l_pocket = /obj/item/pinpointer/nuke/syndicate
+	r_pocket = /obj/item/kitchen/knife/combat
```
Comment (Carbonhell) [inline code/game/gamemodes/nuclear/nuclear.dm]: remove this line, it's no longer needed since you modularized it

**Review by Carbonhell — DISMISSED** [review]

**Comment by Manly-Dwarf** [comment]
do my +1s matter? I dunno, I'll throw one out any way.

**Review by JohnGinnane — DISMISSED** [review]
Code looks good, just waiting for testing to be finished

**Comment by ExpAntique** [comment]
99% sure this is ready to be merged

**Review by JohnGinnane — APPROVED** [review]
Can de get head input

**Comment by JohnGinnane** [comment]
@HippieStation/head

**Comment by Pyko1** [comment]
sure +1

**Pull request merged**